test/csuite/CMakeLists.txt

```diff
@@ -132,13 +132,15 @@ define_c_test(
     DIR_NAME wt2403_lsm_workload
     ARGUMENTS -h $<SHELL_PATH:$<TARGET_FILE_DIR:test_wt2403_lsm_workload>/WT_HOME>
     DEPENDS "WT_POSIX"
+    LABEL "long_running"
```

Comment:
What do you think about a `tsan_long_running` label? These tests are all under 10 minutes on normal builds so they don't qualify as long running.
Similarly for test_fops

Reply:
I don't have much knowledge about the `long_running` label itself, I would be happy to have another one but maybe. 'long_running_sanitizer' as we have some asan ones too. What does the label do though?

Reply:
We created it to ensure our PR testing completes in under 30 minutes. The long tests will then run during the merge queue.

These tags are getting a bit complex though. We also have the concept of [`extra-long`](https://github.com/wiredtiger/wiredtiger/commit/0e8b8669bb909c4ea180ca59916e9d21ad49e137) in the python tests so our categories are now `normal`, `long`, `extra_long`, `sanitizer_long`and two different mechanisms to handle it across python and ctest. 
I'll think on this. CC @lukech in case you have opinions.

Reply:
`long` is currently mapped into the pre-merge testing (or merge queue) scope, and `extra_long` for post-merge testing (or mainline builds). I believe `sanitizer_long` is for a similar purpose of controling runtime for a given testing layer. We don't have these documented right now and probably should.

Reply:
SGTM. We'll add `sanitizer_long` in this PR and I'll create a Tech Debt ticket for documentation

Reply:
`sanitizer_long` added in [97172a3](https://github.com/wiredtiger/wiredtiger/pull/10812/commits/97172a3339b1e9dbb849362f46e9069dbb633116)